spec/support/factory_girl.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.